docs/admin/release_notes/version_2.6.md

```diff
@@ -23,3 +23,24 @@
 
 - [#418](https://github.com/nautobot/nautobot-app-ssot/issues/418) - Unpins multiple dependencies.
 - [#421](https://github.com/nautobot/nautobot-app-ssot/issues/421) - Opened prometheus-client dependency range and removed direct drf-spectacular dependency.
+
+## [v2.6.1 (2024-04-29)](https://github.com/nautobot/nautobot-app-ssot/releases/tag/v2.6.1)
+
+### Added
+
+- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Added additional unit tests for Device42 integration.
+
+### Changed
+
+- [#437](https://github.com/nautobot/nautobot-app-ssot/issues/437) - Changed the Infoblox `utils.client` to make API calls using `requests.Session` instead of `requests.request`.
```

Comment:
```suggestion
- [#437](https://github.com/nautobot/nautobot-app-ssot/issues/437) - Improved performance of the Infoblox client by using `requests.Session` for API calls instead of `requests.request`.
```